### DISCUSSION TITLE
Octo keymaps missing?

### DISCUSSION THREAD

**Discussion by pogopaule** [LazyVim/LazyVim#4547] [Q&A] [answered]
Why do the [keymaps only contain the groups](https://github.com/LazyVim/LazyVim/blob/13a4a84e3485a36e64055365665a45dc82b6bf71/lua/lazyvim/plugins/extras/util/octo.lua#L25-L31)?

**Comment by dpetka2001** [comment]
Because the rest of keymaps are defined by Octo itself. LazyVim just defines the groups, so the keymaps that Octo defines are grouped when `which-key` menu is triggered.

**Reply by pogopaule** [reply]
Ok thanks. Then it seems to be a bug on my machine since it doesn't work. Will investigate...

**Reply by dpetka2001** [reply]
You have to open a Octo buffer for the keymaps to register. So, do like `:Octo issue list`, press `<CR>` to open the issue you like and then press `<leader>ca` to add a comment for example.

**Reply by pogopaule** [reply]
That is what I did, but somehow none of the keymaps defined by octo.nvim are present. I'll try to replicate the issue with a clean installation of LazyVim tomorrow.

**Reply by dpetka2001** [reply]
It works for me just fine with default LazyVim installation.

**Reply by pogopaule** [reply, marked as answer]
Found the issue. [Octo doesn't use `<leader>` but `<space>`](https://github.com/pwntester/octo.nvim/blob/29d4ed4463ce4f487a3d4d0b5a649ab943e62fae/lua/octo/config.lua#L164) :(. I remapped the leaderkey. I Will open a PR there. Thanks for your help.

**Reply by pogopaule** [reply]
created https://github.com/pwntester/octo.nvim/pull/627
